File: src-Kotlin/SelectionSort.kt
import java.util.Scanner

class SelectionSort {

    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        System.out.print("Enter the number of elements : ")
        val n = sc.nextInt()
        val arr = IntArray(n)
        System.out.println("Enter $n elements :")
        for (i in 0 until n)
            arr[i] = sc.nextInt()

        selectionSort(arr)

        System.out.println("\nThe sorted array : ;")
        for (i in 0 until n)
            System.out.print(arr[i].toString() + " ")
        System.out.println()
    }

    internal fun selectionSort(arr: IntArray) {
        val len = arr.size
        var pos: Int
        var tmp: Int
        for (i in 0 until len) {
            pos = i
            for (j in i + 1 until len) {
                if (arr[j] < arr[pos])
                    pos = j
            }
            tmp = arr[i]
            arr[i] = arr[pos]
            arr[pos] = tmp
        }
    }
}

File: Php/SelectionSort.php
<?php 
    $arr = array(5,3,1,5,6,2,3);
    for($i=0;$i<count($arr);$i++){
        echo $arr[$i];
    }
    echo "-";
    $arr2 = selectionSort($arr);
    for($i=0;$i<count($arr2);$i++){
        echo $arr2[$i];
    }

    function selectionSort($arr){
        for($i = 0; $i< count($arr);$i++){
            $pos = $i;
            for ($j = $i + 1; $j < count($arr); $j++) {
                if ($arr[$j] < $arr[$pos])
                    $pos = $j;
            }
            $tmp = $arr[$i];
            $arr[$i] = $arr[$pos];
            $arr[$pos] = $tmp;
        }

        return $arr;
    }
?>